lib/dl_db_testing/dl_db_testing/database/base.py

```diff
@@ -158,15 +158,22 @@ class DbTableBase:
     def name(self) -> str:
         return self.table.name
 
-    def insert(self, data: Union[dict, List[dict]]) -> None:
+    def insert(self, data: Union[dict, List[dict]], bulk_insert: bool = False, chunk_size: int = 200) -> None:
```

Comment:
`bulk_insert=False` implies `chunk_size=1`, so maybe one new argument is enough?

Reply:
well, ok...
I thought about it, but it didn't seem very user-friendly to me